.gitignore

```diff
@@ -4,19 +4,18 @@ tests/data/*.json
 tests/pdal/test_pdalc_*.c
 !tests/pdal/test_pdalc_utils.c
 
-# Ignore doxygen docs
-/doc/doxygen
-/doc/Doxyfile
-
```

Comment:
What was the rationale for including Doxygen docs in the repository?

Reply:
So that they can be published through GH pages